R/process-KNMI-netCDF-files.R

```diff
@@ -0,0 +1,100 @@
+# Function to process individual netCDF files ####
+
+# Author: Cherine Jantzen
+# Created: 2024-02-21
+# Last updated: 2024-02-22
+
+# I. Load packages --------------------------------------------------------
+
+library(tidync)
+library(ncmeta)
+library(RNetCDF)
+library(dplyr)
+library(purrr)
+library(tidyr)
+library(here)
+
+# II. Function ------------------------------------------------------------
+
+# Arguments: 
+# - bbox: Spatial bounding box for which to retrieve data. Vector of four numeric values, indicating western-most, southern-most, eastern-most and northern-most point of the bounding box (in Amersfoort/RD New (ESPG:28992)).
+# - netCDF_location: Character specifying the folder name in which the netCDF files are stored. Folder needs to be a sub-folder of the working directory. 
+
+process_netCDF <- function(bbox,
+                           netCDF_location){
+  
+  
+  # set longitude and latitude range according to boundng box
+  lonrange <- c(bbox[1], bbox[3])
+  latrange <- c(bbox[2], bbox[4])
```

Comment:
Did we discuss whether we wanted to include a test here? To make sure that the coordinates are in the right CRS and in the right order?